#! /bin/sh /usr/share/dpatch/dpatch-run
## 02_add_to_proxy_group.dpatch by Javier Uruen Val <javi@warp.es>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Patch to backport 0.11.100 fix to add run eBox under proxy group

@DPATCH@

Index: conf/80squid.conf
===================================================================
--- squid/conf/80squid.conf	(.../ubuntu-hardy-0.11.99/client/squid)	(revisión: 9796)
+++ squid/conf/80squid.conf	(.../ubuntu-0.11.99/client/squid)	(revisión: 9796)
@@ -18,4 +18,4 @@
 # cache_mem [required]. Amount of memory to be used by squid 
 cache_mem = 100 
 
-
+group = proxy
